### DISCUSSION TITLE
Channels DVR moved perfectly

### DISCUSSION THREAD

**Discussion by hausmanconsulting** [007revad/Synology_app_mover#76] [General]
I used version 3.0.53 to move the Channels DVR server from an HDD volume to an SSD/NVMe volume after backing up.

I did get one notice after the backup:

`/usr/syno/sbin/synoshare: unrecognized option '--get-real-path'`

but it seems to have worked (didn't have to restore though).

The move worked fine as far as I can tell; I've run the web portal and run the app on a TV client and it works nicely, much quicker than before. Awesome work on this!

**Comment by 007revad** [comment]
Which DSM version are you using?

**Reply by 007revad** [reply]
I've worked out the `synoshare --get-real-path` option was added in DSM 7.2. I had mistakenly though it was added in 7.0.

I've released an updated version of the script that uses `synoshare  --get-map` for any DSM version less than 7.2

**Reply by hausmanconsulting** [reply]
I'm on DSM 7.2-64570 Update 3 on a DS918+.

**Reply by 007revad** [reply]
Ahh, so the `synoshare --get-real-path` option was added in DSM 7.2.1

**Comment by 007revad** [comment]
I've added Channels DVR to the "Packages confirmed working" list.
